### DISCUSSION TITLE
Unable to install extension 'ms-ceintl.vscode-language-pack-zh-hans' as it is not compatible with VS Code '1.71.2'.

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#5663] [Q&A]
Unable to install extension 'ms-ceintl.vscode-language-pack-zh-hans' as it is not compatible with VS Code '1.71.2'.
<img width="910" alt="image" src="https://user-images.githubusercontent.com/9245110/196391455-081afa07-e6a8-46c7-a3d9-e00e99965933.png">

but install ok from marketspace?

<img width="1138" alt="image" src="https://user-images.githubusercontent.com/9245110/196392381-988aa344-2a4f-4c4a-be25-a7eee310ef1b.png">

and I can't see installed extension version info?

**Comment by jsjoeio** [comment]
Hmm 🤔 There was a bug with that happening on the command line where it would say not compatible. Maybe it still exists? But yes, installing from marketplace sounds like it works? I'm not sure why it's not showing the version info

**Reply by alanhe421** [reply]
The background is that I want to deploy code-server with one click and initialize it to the target locale.


This is confusing to me.

**Reply by jsjoeio** [reply]
We have some one click deploys [here](https://github.com/coder/deploy-code-server). For initializing in the target locale, take a look at our e2e test [here](https://github.com/coder/code-server/blob/main/test/e2e/displayLang.test.ts#L4-L8). Basically what you'll need to do is:
- download the [extension language pack](https://github.com/coder/code-server/tree/main/test/e2e/extensions/ms-ceintl.vscode-language-pack-es-1.70.0)
- add the [`languagepacks.json`](https://github.com/coder/code-server/blob/main/test/e2e/models/CodeServer.ts#L90-L111)
- run code-server with locale flag i.e. `--locale es` for Spanish

Ideally we fix the root cause but we haven't had any bandwidth to do so unfortunately.

**Reply by alanhe421** [reply]
it's ok. it meet my needs.

but I have a question.


code-server cli support  locale argv?

I cannot see any desc when type `--help`.


![image](https://user-images.githubusercontent.com/9245110/196723281-990e3b7e-f77e-406f-8bb3-e17cf14f14bc.png)

**Reply by jsjoeio** [reply]
Yes, it does! I'm not sure why it doesn't show in `--help` but it is supported because it's supported by VS Code.
